authoidc.properties

```diff
@@ -1,14 +1,20 @@
 category=Integration
 description=Delegate authentication using the OpenID Connect protocol
 homepageUrl=https://github.com/vaulttec/sonar-auth-oidc
-archivedVersions=1.0.1,1.0.2,1.0.3,1.0.4
-publicVersions=1.1.0
+archivedVersions=1.0.1,1.0.2,1.0.3,1.0.4,1.1.0
+publicVersions=2.0.0
 
 defaults.mavenGroupId=org.vaulttec.sonarqube.auth.oidc
 defaults.mavenArtifactId=sonar-auth-oidc-plugin
 
+2.0.0.description=Breaking changes: user's login is always auto-generated by SonarQube, provider metadata is retrieved from the provider and the list of OAuth scopes is fully configurable
+2.0.0.sqVersions=[7.4,LATEST]
```

Comment:
This range needs to start at 7.9, the oldest version still supported by the Marketplace.

Reply:
fixed